messenger/seqclient.go

```diff
@@ -44,6 +55,20 @@ func (sc *SequencerClient) broadcastTxSync(tx *astriaPb.SignedTransaction) (*ten
 	return sc.c.BroadcastTxSync(context.Background(), tx)
 }
 
+func (sc *SequencerClient) SendMessageViaComposer(tx []byte) error {
+	grpcCollectorServiceClient := composerv1alpha1grpc.NewGrpcCollectorServiceClient(sc.composerClient)
+	// if the request succeeds, then an empty response will be returned which can be ignored for now
+	_, err := grpcCollectorServiceClient.SubmitRollupTransaction(context.Background(), &astriaComposerPb.SubmitRollupTransactionRequest{
+		RollupId: sc.rollupId,
+		Data:     tx,
+	})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
 // SendMessage sends a message as a transaction.
 func (sc *SequencerClient) SendMessage(tx []byte) (*tendermintPb.ResultBroadcastTx, error) {
```

Comment:
can remove this if no longer used

Reply:
umm i thought, we could just have it for now as a fallback or so? like i didn't want to completely remove that code cause what if we want some fallback logic in case composer fails right.